lib/styles.js

```diff
@@ -350,18 +350,37 @@ module.exports = function (expect) {
         }
     });
 
-    expect.addStyle('diffFragment', function (text, baseStyle, specialCharStyle, markUpSpecialCharacters) {
-        if (markUpSpecialCharacters) {
-            text.split(specialCharRegExp).forEach(function (part) {
-                if (specialCharRegExp.test(part)) {
-                    this[specialCharStyle || baseStyle](utils.escapeChar(part));
-                } else {
-                    this[baseStyle](part);
-                }
-            }, this);
-        } else {
-            this[baseStyle](text);
-        }
+    expect.addStyle('diffFragment', function (type, text, baseStyle, specialCharStyle, markUpSpecialCharacters) {
+        var lines = text.split(/\n/);
+        lines.forEach(function (line, i) {
+            // Are we at the start of a line? (might want to add a magicpen method for this :)
```

Comment:
@sunesimonsen Would you be OK with me adding a `isAtStartOfLine` method to magicpen?